README.md

```diff
@@ -248,6 +248,15 @@ Take a look at `BTCardFormViewController.m` to see examples of using the form fi
 
 Start with [**'Hello, Client!'**](https://developers.braintreepayments.com/ios/start/hello-client) for instructions on basic setup and usage.
 
+## Versions
+
+This SDK abides by our Client SDK Deprecation Policy. For more information on the potential statuses of an SDK check our [developer docs](http://developers.braintreepayments.com/guides/client-sdk/deprecation-policy).
+
+| Major version number | Status | Released | Deprecated | Unsupported |
+| -------------------- | ------ | -------- | ---------- | ----------- |
+| 9.x.x | Beta | March 2021 | TBA | TBA |
```

Comment:
```suggestion
| 9.x.x | Beta | TBA | TBA | TBA |
```

Should we just put TBA for the v9 release date? Or omit it for now and add it when we actually do a beta release?